resources/assets/js/components/SearchBar.tsx

```diff
@@ -2,21 +2,19 @@ import * as React from "react";
 import { Formik, Form, FastField } from "formik";
 import * as Yup from "yup";
 import { useIntl, defineMessages } from "react-intl";
-import SkillSearchResults, { SkillSearchResult } from "./SkillSearchResults";
-import { validationMessages } from "../../Form/Messages";
-import { getLocale } from "../../../helpers/localize";
-import TextInput from "../../Form/TextInput";
-import { Skill } from "../../../models/types";
+import { validationMessages } from "./Form/Messages";
+import { getLocale } from "../helpers/localize";
+import TextInput from "./Form/TextInput";
 
 const formMessages = defineMessages({
   searchLabel: {
     id: "searchBar.search.label",
-    defaultMessage: "Search",
+    defaultMessage: "Search for skills by name:",
     description: "The label displayed for the search input field.",
   },
   searchPlaceholder: {
     id: "searchBar.search.placeholder",
-    defaultMessage: "eg. Enter a value.",
+    defaultMessage: "eg. User interface design.",
     description: "The placeholder displayed for the search input field.",
   },
```

Comment:
I see you've moved this out of the ApplicantProfile directory because its _so close_ to being a totally generic search bar component. These strings are still specific to Skill search, so maybe pass them in as props?